app/models/job_invocation.rb

```diff
@@ -48,7 +48,7 @@ class JobInvocation < ActiveRecord::Base
 
   scope :with_task, -> { references(:task) }
 
-  scoped_search :in => :recurring_logic, :on => 'id', :rename => 'recurring_logic.id', :auto_complete => true
+  scoped_search :in => :recurring_logic, :on => 'id', :rename => 'recurring_logic.id', :complete_value => true
```

Comment:
I don't think it's exposed anywhere in UI but I see your point. I think it's safe to remove it. I'll merge this PR as it is to see that tests becomes green and then open separate issue.